### PR TITLE
Screen constants

### PR DESCRIPTION
**Pull request by IotaBread**
Maps most remaining screen constants, and a few other methods

### CODE REVIEW

**Review by LambdAurora — APPROVED** [review]

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/client/gui/screen/Screen.mapping [hunk 1]
```diff
@@ -8,12 +8,16 @@ CLASS net/minecraft/unmapped/C_wrmtlwqx net/minecraft/client/gui/screen/Screen
 	FIELD f_htifmqlv title Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_kmcqeiie width I
 	FIELD f_kreiluct children Ljava/util/List;
+	FIELD f_lpsgbjcj EXTRA_SPACE_AFTER_FIRST_TOOLTIP_LINE I
```
Comment (Bubblie01) [inline mappings/net/minecraft/client/gui/screen/Screen.mapping]: Maybe take extra out?

**Review by Bubblie01 — APPROVED** [review]

File: mappings/net/minecraft/client/gui/screen/Screen.mapping (hunk 1 above)
Comment (Jamalam360) [reply mappings/net/minecraft/client/gui/screen/Screen.mapping]: I agree this is quite long and would probably be fine without extra

**Review by Jamalam360 — COMMENTED** [review]
I sent a question in the internal mappings channel, sorry probably should have added that as a review comment instead :/. I'd imagine it's fine I'm just double checking

**Review by Jamalam360 — APPROVED** [review]
The question in the internal mappings channel was resolved. I'm approving now because I'll be busy tomorrow, but Bubblies question is still there.

**Review by NoComment1105 — APPROVED** [review]
All good, I agree with Bubblie on his comment, so that's the only thing I've got

**Pull request merged**